community/mypaint-brushes/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Sascha Paunovic <azarus@posteo.net>
+# Maintainer: Sascha Paunovic <azarus@posteo.net>
+pkgname=mypaint-brushes
+pkgver=1.3.0
+pkgrel=0
+pkgdesc="default MyPaint brushes"
+url="https://github.com/Jehan/mypaint-brushes"
+arch="noarch"
+license="CC0-1.0"
+makedepends="autoconf automake"
+source="$pkgname-$pkgver.tar.gz::https://github.com/Jehan/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/"$pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	autoreconf -i
```

Comment:
This should preferably be done in `prepare()`

Reply:
Both changes are now in the rebased and edited commits.